terminal/terminal.md

```diff
@@ -0,0 +1,417 @@
+# The amazing world of the Terminal
+
+The terminal is a command-line interface app to control the operating system with text based commands
+
+To understand what the terminal is we need to ask ourselves, if the user interface in the computer would not exist how we would be able to operate it normally as we do? move across the directories and editing files? 
+
+The terminal emulator also known as just 'Terminal' is a programs created to display the computer's data, the term comes from when users used to connect through a hardware called terminals that were only a display and a keyboard connected to a main computer called mainframe with a specific user who had specific permissions
```

Comment:
+The terminal emulator also known as just 'Terminal' is a program~s~ created to display the computer's data, the term comes from when users used to connect through a hardware called terminals that were only a display and a keyboard connected to a main computer called mainframe with a specific user who had specific permissions`.`